overlays/emacs.nix

```diff
@@ -16,7 +16,7 @@ let
       super.emacs
       ([
 
-        (drv: drv.override ({ srcRepo = true; } // args))
+        (drv: drv.override ({ srcRepo = true; withXwidgets = false; } // args))
```

Comment:
I think only Emacsens using `emacs-master.json` are affected by that patch.  But I think it is fine to change all Emacsens.